portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,9 +28,29 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
```

Comment:
can we change the servlet name to be slightly more specific? I think "dataservlet" might get confusing with all the other servlets